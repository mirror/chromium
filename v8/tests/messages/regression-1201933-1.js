// This is okay.
function foo() { const a; var a; }

// This isn't.
(function () {
  const a;
  var a;
})();
