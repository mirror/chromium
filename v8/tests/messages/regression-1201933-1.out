regression-1201933-1.js:7: Uncaught TypeError: const 'a' has already been declared
  var a;
      ^
